docs/introduction.rst

```diff
@@ -0,0 +1,42 @@
+What you need to do to start using ROSS
+=======================================
+
+.. _introduction:
+
+Programming language: Python
+----------------------------
+
+ROSS is an application that runs in Python, a high level programming language.
+If you don't have Python installed in your machine, the first thing you should do is install it.
+Refer to the `Python website
+<http://www.python.org/>`_ to do so. ROSS's code was tested in Python 3.6 and higher, so make sure you
```

Comment:
Since ross requires packages such as numpy / scipy, which are not easy to compile, I think we should recommend a distribution such as [anaconda](https://www.anaconda.com/distribution/).